src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -175,45 +175,93 @@
  * <pre>
  * &lt;module name=&quot;DesignForExtension&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public abstract class Foo {
+ *   private int bar;
+ *
+ *   public int method1() {return 2;}  // Violation. No javadoc.
+ *
+ *   public int method2() {return 8;}  // Violation. No javadoc.
+ *
+ *   private void method3() {method4();}  // OK. Private method.
+ *
+ *   protected void method4() { }  // OK. No implementation.
+ *
+ *   public abstract void method5();  // OK. Abstract method.
+ *
+ *   &#47;**
+ *    * This implementation ...
+ *    &#64;return some int value.
+ *    *&#47;
+ *   public int method6() {return 1;}  // OK. Have javadoc on overridable method.
+ *
+ *   &#47;**
+ *    * Some comments ...
+ *    *&#47;
+ *   public int method7() {return 1;}  // OK. Have javadoc on overridable method.
+ *
+ *   &#47;**
+ *    * This
+ *    * implementation ...
+ *    *&#47;
+ *   public int method8() {return 2;}  // OK. Have javadoc on overridable method
+ *
+ *   &#64;Override
+ *   public String toString() {  // Violation. No javadoc for @Override method
+ *     return "Foo{" +
```

Comment:
`return "";`

Reply:
Thank you! I will do that as soon as I get home!

Reply:
Hi @romani , I already resolve this one in commit [184f6da](https://github.com/zwwang98/checkstyle/commit/184f6da3f50b2d5e26cc0765fd165a23ae29f7e8).

Reply:
done